src/internal/tester/suite.go

```diff
@@ -0,0 +1,39 @@
+package tester
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/suite"
+)
+
+func NewUnitSuite(t *testing.T, excludeGroups ...string) *UnitSuite {
+	dontRunOn(
+		t,
+		append(
+			[]string{CorsoIntegrationTests},
+			excludeGroups...,
+		)...,
+	)
```

Comment:
This seems extraneous.  Unit testing should only take a  couple seconds.  Weeding it out of integration tests kinda feels like throwing a single cup of water out of a pool.

If we hit a point in CI design where we want to separately run one pass on unit tests and another pass on integration tests, that's the time to add something like this.  But at the moment I don't see much use in it.

Reply:
sure, that's fine. I wasn't sure if we wanted them completely separate or not. Easy enough to make them all run and remove the extra env var I added